Create Intro to Calculating Cred and Grain

s-ben opened the pull request:
This PR is for @miyazono's article "An introduction to calculating cred and grain". 

@decentralion asked me to squash commits from this [PR](https://github.com/sourcecred/docs/pull/28), which itself branched off from this [PR](https://github.com/sourcecred/docs/pull/25,) so that we could have a clean PR to merge. Note that I contributed much less to this than @miyazono, @decentralion, and @vsoch. This should be OK, as we're planning to turn this article into an Initiative anyway, which will flow an appropriate amount of cred to @miyazono. Also @'ing all the contributors I see on the other PRs and adding "paired with @miyazono" to mitigate any excess cred I might get for submitting this PR.

decentralion commented:
Please remove the file `community/concepts/grain.md` from this pull--it's superseded by #30. (That's the main diff I wanted between this and #28.)

s-ben commented:
Removed `community/concepts/grain.md`, squashed commit. Deleting `grain.md` seems to have also deleted the `/concepts` for some reason though...am I doing this right?

decentralion commented:
Yeah, you did it right. Having removed grain.md, you won't see the concepts/ directory locally because this pr is based off an old version of master that doesn't have any of the glossary yet. So without this PR adding a glossary concept, the directory just doesn't exist.

However, that's fine--other PRs have since added the glossary, and merged, so when this PR merges (to the current master branch) those commits will be present. Also, this commit won't create merge conflicts since it is only creating new content, so it will be easy to squash and merge-- I'll do so presently.
